### PR TITLE
OCM-3249 | chore: update cluster pattern in test

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

**Comment by willkutler** [comment]
LGTM

**Review by ciaranRoche — COMMENTED** [review]
Nice improvement, just a small comment inline

File: cmd/verify/network/cmd_test.go [hunk 1]
```diff
@@ -393,16 +152,23 @@ rosa verify network --watch --status-only --region us-east-1 --subnet-ids subnet
...
-			clusterJson["items"].([]interface{})[0].(map[string]interface{})["status"].(map[string]interface{})["state"] = state
-			updatedCluster, _ := json.Marshal(clusterJson)
+			mockCluster, err := test.MockOCMCluster(func(c *cmv1.ClusterBuilder) {
+				awsBuilder := cmv1.NewAWS()
+				awsBuilder.SubnetIDs("subnet-0b761d44d3d9a4663", "subnet-0f87f640e56934cbc")
+				c.AWS(awsBuilder)
+				regionBuilder := cmv1.NewCloudRegion()
+				regionBuilder.ID("us-east-1")
+				c.Region(regionBuilder)
+				c.State(state)
```
[the remark this answers is not in the document]
Comment (ciaranRoche) [reply cmd/verify/network/cmd_test.go:158]: @thomasmckay just bumping this, it says you pushed a changed, but I am not seeing it :thinking:

[the remark this answers is not in the document]
Comment (thomasmckay) [reply cmd/verify/network/cmd_test.go]: oops, i only changed it in one place. corrected

**Review by ciaranRoche — COMMENTED** [review]
/lgtm

**Pull request merged**